### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by martin12333** [martin12333/browserPOSIX-discussion#16] [General]
Please introduce yourself.

**Comment by martin12333** [comment]
Ok, so I will start with a mini introduction of myself:

One of my dream projects is something like: Unix tools in wasm.

But I will have to learn a lot.

I am glad, that I was able to install cowasm dash and utils locally, and also compile something
(my experiments are at
https://github.com/martin12333/marti-onedrive/blob/main2/cowasm/
)

**Comment by guest271314** [comment]
Hi.
